content.js

```diff
@@ -0,0 +1,106 @@
+(function() {
```

Comment:
This file has a few small nitpicks I'll summarize here:

- `===` on line 33, 87
- need semicolon on line 101